docs/modules/ROOT/pages/delete-account.adoc

```diff
@@ -0,0 +1,45 @@
+= Deleting an Account
+:description: pass:q[If you no longer need to access your clusters, you can delete your account. If want to avoid additional charges while your clusters are not in use, consider xref:stop-and-resume.adoc[pausing] them instead.]
+:cloud-tags: Manage Accounts
+:cloud-title: Deleting an Account
+:cloud-order: 60
+:toclevels: 3
+
+{description}
+
+
+[[cluster-and-data-deletion]]
+== Cluster and Data Deletion
+
+Deleting an account _removes_ all of the following:
+
+- Clusters linked to the account, including any associated data, such as data structures, custom classes, backups, and configuration.
+- Account and payment details, excluding your email address.
```

Comment:
Do we need to provide a reason why do we keep the email?

Reply:
I've reworded and added a note. I've tried to separate out the retention of the users email address (in line with our privacy policy) and the deletion of the account.